.github/pull_request_template.md

```diff
@@ -17,6 +17,7 @@ Before merging, the reviewer should check the following items:
 - [ ] Does the PR do what it claims to do?
 - [ ] If this is a new adaptor, has the migration tool been run and the
       migration guide followed?
+- [ ] If this is a new adaptor, Added the adaptor ot marketing website ?
```

Comment:
Typo on `ot`

Can we remove the previous `if this is a new adaptor` line? It's irrelevant now

Reply:
Thanks @josephjclark i have fixed the typo and remove migration checklist